docs/lib/graphqlapi/fragments/graphql-from-node.md

```diff
@@ -0,0 +1,86 @@
+You can also call an AppSync GraphQL API from a NodeJS app or a Lambda function. For example, deploy a simple `Todo` model with the following schema in the `amplify add api` flow:
```

Comment:
@mlabieniec / @undefobj  What do you think about adding this, we are pretty explicit about not having Node callouts in Amplify. This would be new.

Reply:
We have Node callout in the existing docs which is what Nader is porting over, not a new thing. What we're explicit is that Amplify JS isn't supported in Node. The multi-auth feature was actually added for this reason.

Reply:
Do you think I should also add an example of calling with an API key?